### PR TITLE
Add DeepSpeed BERT Squad QA test

### PR DESCRIPTION
**Pull request by rsarm**

### CODE REVIEW

File: checks/apps/deeplearning/deepspeed/check_deepspeed_bert.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import reframe as rfm
+import reframe.utility.sanity as sn
+from reframe.core.backends import getlauncher
+
+
+class deepspeed_bert_fetch_data_tokenizer(rfm.RunOnlyRegressionTest):
+    '''Fixture for fetching the the dataset and tokenizer'''
+    local = True
+    modules = ['PyTorch']
+    prerun_cmds = ['. $SCRATCH/deeepspeed-env/bin/activate']
```
Comment (mszpindler) [inline checks/apps/deeplearning/deepspeed/check_deepspeed_bert.py:11]: I am not sure if I understand how the fixture method works. Is this virtualenv already exists in the container?

File: checks/apps/deeplearning/deepspeed/check_deepspeed_bert.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    @performance_function('samples/sec')
+    def samples_per_sec(self):
+        return sn.avg(sn.extractall(r'SamplesPerSec=(?P<samples_per_sec>\S+),',
+                                    self.stdout, 'samples_per_sec', float))
+
+
+@rfm.simple_test
+class deepspeed_bert_qa_train(deepspeed_bert_qa_train_base):
+    modules = ['PyTorch']
+    prerun_cmds = ['module unload aws-ofi-rccl',
```
Comment (mszpindler) [inline checks/apps/deeplearning/deepspeed/check_deepspeed_bert.py:58]: Why the rccl module is unloaded here?

**Review by mszpindler — COMMENTED** [review]
I do not have enough knowledge to review the Bert scripts. Besides the fixture thing I do not understand here, all looking good.

**Comment by rsarm** [comment]
Hi @mszpindler Just tested this, it's ready. Could you have a look?

**Review by mszpindler — APPROVED** [review]

**Pull request merged**